paddle/fluid/inference/tests/api/CMakeLists.txt

```diff
@@ -160,6 +160,12 @@ if(WITH_MKLDNN)
   set(INT8_DATA_DIR "${INFERENCE_DEMO_INSTALL_DIR}/int8v2")
   if (NOT EXISTS ${INT8_DATA_DIR})
     inference_download_and_uncompress(${INT8_DATA_DIR} "${INFERENCE_URL}/int8" "imagenet_val_100_tail.tar.gz")
+    inference_download_and_uncompress(${INT8_DATA_DIR} "${INFERENCE_URL}/int8" "pascalvoc_val_100_head.tar.gz")
+  endif()
```

Comment:
please do not put data here. since `if (NOT EXISTS ${INT8_DATA_DIR})` is false on CI

Reply:
> please do not put data here. since `if (NOT EXISTS ${INT8_DATA_DIR})` is false on CI

Aha! ok I see, because the directory exists already. Thank you for advice!

Reply:
Do not review now. Tomorrow, I will 
1. squash and force-push. [Because I changed most of the python script]
2. Change some user-interface descriptions in converting process in full_pascalvoc.py
3. Get inner reviews.

Reply:
@wojtuss  @sfraczek  Hi! Please review when you have time. Thank you !